x/ibc/07-tendermint/client/cli/tx.go

```
@@ -111,13 +113,23 @@ func NewCreateClientCmd() *cobra.Command {
 				}
 			}
 
+			allowUpdateAfterExpiry, err := cmd.Flags().GetBool(flagAllowUpdateAfterExpiry)
+			if err != nil {
+				return err
+			}
+			allowUpdateAfterMisbehaviour, err := cmd.Flags().GetBool(flagAllowUpdateAfterMisbehaviour)
+			if err != nil {
+				return err
+			}
```

nit, you can ignore the errors, the value will default to false